### DISCUSSION TITLE
FYI: theia-cloud and theia-cloud-helm repositories will be moved to the eclipse-theia organization

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia-cloud#351] [Announcements]
Via the temporary eclipse-theia-cloud-trasnfer org.

**Comment by Valkovets** [comment]
could you please specify what i should use instead of https://github.eclipsesource.com/theia-cloud-helm?

**Reply by sgraband** [reply]
The new repositories are:
- https://github.com/eclipse-theia/theia-cloud/
- https://github.com/eclipse-theia/theia-cloud-helm/

But the old links should simply redirect. You have a typo in the link above. It's not https://github.eclipsesource.com/theia-cloud-helm, but rather https://github.com/eclipsesource/theia-cloud-helm (Note that this redirects to https://github.com/eclipse-theia/theia-cloud-helm/)

**Reply by Valkovets** [reply]
thanks, I taked it from  https://main--theia-cloud.netlify.app/documentation/setuptheiacloud/ 
![image](https://github.com/user-attachments/assets/ff9a2000-dec8-4ac3-9011-601471ecc8e3)

**Reply by sgraband** [reply]
Ah sorry you were talking about the url for the helm-repo. My Mistake. The new url is:

https://eclipse-theia.github.io/theia-cloud-helm/

I will update the documentation accordingly. Thank you for raising this.
